ApplicationInsightsRequestLogging/ApplicationInsightsRequestLogging/Options/BodyLoggerOptions.cs

```diff
@@ -45,5 +45,23 @@ public BodyLoggerOptions()
         ///     Defines the text to append in case the body should be truncated <seealso cref="MaxBytes"/>
         /// </summary>
         public string Appendix { get; set; } = "\n---8<------------------------\nTRUNCATED DUE TO MAXBYTES LIMIT";
+
+        public IEnumerable<string> PropertyNamesWithSensitiveData { get; set; } = new List<string>()
```

Comment:
```suggestion
        public List<string> PropertyNamesWithSensitiveData { get; set; } = new List<string>()
```

So the user can directly call `Add()` when configuring the middleware.